cli/pkg/kctrl/cmd/package/builder/create.go

```diff
@@ -0,0 +1,450 @@
+		maintainers = append(maintainers, v1alpha1.Maintainer{Name: maintainerName})
+	}
+	create.Maintainers = maintainers
+	return nil
+}
+
+func (create *CreateStep) configureFetchSection() error {
+	fetchConfiguration := fetch.NewFetchStep(create.Ui, create.PkgLocation)
+	err := fetchConfiguration.Run()
+	if err != nil {
+		return err
+	}
+	create.Fetch = *fetchConfiguration
+	return nil
+}
+
+func (create *CreateStep) configureTemplateSection() error {
+	templateConfiguration := template.NewTemplateStep(create.Ui)
+	err := templateConfiguration.Run()
+	if err != nil {
+		return err
+	}
+	create.Template = *templateConfiguration
+	return nil
+}
+
+func (create *CreateStep) configureValuesSchema() error {
+	valuesSchema, err := create.getValueSchema()
+	if err != nil {
+		return err
+	}
+	create.ValuesSchema = valuesSchema
+	return nil
+}
+
+func (create CreateStep) getValueSchema() (v1alpha1.ValuesSchema, error) {
+	valuesSchema := v1alpha1.ValuesSchema{}
+	var isValueSchemaSpecified bool
+	var isValidInput bool
+	input, err := create.Ui.AskForText("Do you want to specify the values Schema(y/n)")
+	if err != nil {
+		return valuesSchema, err
+	}
+	for {
+		isValueSchemaSpecified, isValidInput = common.ValidateInputYesOrNo(input)
+		if !isValidInput {
+			input, err = create.Ui.AskForText("Invalid input. (must be 'y','n','Y','N')")
+			if err != nil {
+				return valuesSchema, err
+			}
+			continue
+		}
+		if isValueSchemaSpecified {
+			valuesSchemaFileLocation, err := create.Ui.AskForText("Enter the values schema file location")
+			if err != nil {
+				return valuesSchema, err
+			}
+			valuesSchemaData, err := readDataFromFile(valuesSchemaFileLocation)
+			if err != nil {
+				return valuesSchema, err
+			}
+			valuesSchema = v1alpha1.ValuesSchema{
+				OpenAPIv3: runtime.RawExtension{
+					Raw: valuesSchemaData,
+				},
+			}
+		} else {
+			break
+		}
+	}
+	return valuesSchema, nil
+
+}
+func readDataFromFile(fileLocation string) ([]byte, error) {
+	//TODO should we read it in a buffer
+	data, err := os.ReadFile(fileLocation)
+	if err != nil {
+		return nil, err
+	}
+	return data, nil
+}
+
+func (createStep CreateStep) PostInteract() error {
+	err := createStep.createPackageBuilder()
+	if err != nil {
+		return err
+	}
+	err = createStep.createAndPrintPackageCR()
+	if err != nil {
+		return err
+	}
+	err = createStep.createAndPrintPackageMetadataCR()
+	if err != nil {
+		return err
+	}
+	str := fmt.Sprintf(`
+# Both the files can be accessed from the following location: %s
+`, createStep.PkgLocation)
+	createStep.Ui.PrintBlock([]byte(str))
+
+	return nil
+}
+
+func (create CreateStep) createPackageBuilder() error {
+	pkgBuilder := create.populatePkgBuilder()
+	jsonPackageData, err := json.Marshal(&pkgBuilder)
+	if err != nil {
+		return err
+	}
+	yaml.JSONToYAML(jsonPackageData)
+	packageData, err := yaml.JSONToYAML(jsonPackageData)
+	if err != nil {
+		return err
+	}
+	pkgFileLocation := create.PkgLocation + "/pkgBuilder.yml"
```

Comment:
lets use `package-build.yml` as name (no camel casing in names)

Reply:
Adopted